pkg/sdscomm/sdscomm.go

```diff
@@ -0,0 +1,120 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package sdscomm
```

Comment:
this should be under `pkg/controller/command`

Reply:
Done!

Reply:
Sorry, maybe I wasn't clear enough: the full path should be `pkg/controller/command/sdscomm`

Reply:
Done!